spec/controllers/api/lists_controller_spec.rb

```diff
@@ -0,0 +1,97 @@
+require 'rails_helper'
+
+RSpec.describe ListsController, type: :controller do
+  before do
+    @user = create(:user)
+    @user1 = create(:user)
+    @list = create(:list, user_id: @user.id)
+    @token = JsonWebToken.encode(user_id: @user.id)
+  end
+  
+  describe "GET #show_users_lists" do
+    context "when valid" do
+      before do
+        request.headers["AUTHORIZATION"] = "Bearer #{@token}"
+        get :show_users_lists, format: :json, params: { id: @user.id }
+        @json_response = JSON.parse(response.body)
+      end
+      
+      it "returns http success" do
+        expect(response).to have_http_status(:success)
+      end
+      
+      it "JSON body response contains expected lists attributes" do
+        expect(@json_response.first.keys).to match_array(["created_at", "description", "id", "movies", "name", "public", "updated_at", "user_id"])
+      end
+
+      it "JSON body response contains expected list name" do
+        expect(@json_response.first['name']).to eq('Movies List')
```

Comment:
Instead of writing the attributes (e.g. 'Movies List') why don't you compare it like this? 
```
expect(@json_response.first['name']).to eq(@list.name)
```
Same with examples below.
That way if you change the factory, this test won't fail.